src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/IdentifierNormalizerPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Compiler\PriorityTaggedServiceTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Injects identifier normalizers.
+ *
+ * @internal
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class IdentifierNormalizerPass implements CompilerPassInterface
```

Comment:
Maybe can you bump the dep `symfony/di` to 3.4 and use https://symfony.com/blog/new-in-symfony-3-4-simpler-injection-of-tagged-services instead.

Reply:
Why don't we keep this for another PR where we use this improvement on every cases we have?

Reply:
If we introduce a new class, we need to maintain it (and deprecate it soon), but your call.